mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/ui/Node.kt

```diff
@@ -14,14 +14,12 @@ import com.jakewharton.mosaic.ui.unit.Constraints
 
 @Composable
 @MosaicComposable
-@Suppress("ktlint") // TODO why doesn't "ktlint:compose:param-order-check" work here?
```

Comment:
It didn't work there because `content` had a default value and it was laid out before other parameters that were non-optional. I did a "Change Signature" IJ refactor here, but let me know if you'd rather have it done as before (I looked at the callsites and all were providing modifiers so this looked alright to do).